File: app/src/main/java/org/bitcoindotcom/bchprocessor/bip70/model/InvoiceStatusOutput.java
package org.bitcoindotcom.bchprocessor.bip70.model;

import com.google.gson.annotations.SerializedName;

public class InvoiceStatusOutput {
    @SerializedName("script")
    public String script;
    @SerializedName("amount")
    public long amount;
    @SerializedName("address")
    public String address;
    @SerializedName("type")
    public String type;

    @Override
    public String toString() {
        return "InvoiceStatusOutput{" +
                "script='" + script + '\'' +
                ", amount=" + amount +
                ", address='" + address + '\'' +
                ", type='" + type + '\'' +
                '}';
    }
}

File: app/src/main/java/org/bitcoindotcom/bchprocessor/bip70/model/InvoiceStatusOutput.kt
package org.bitcoindotcom.bchprocessor.bip70.model

import com.google.gson.annotations.SerializedName

data class InvoiceStatusOutput(
        @SerializedName("script") var script: String? = null,
        @SerializedName("amount") var amount: Long = 0,
        @SerializedName("address") var address: String = "",
        @SerializedName("type") var type: String? = null
)